modules/ob-release-notes-1-3.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-3_{context}"]
+= Release notes for {builds-shortname} General Availability 1.3
+
+With this update, {builds-shortname} 1.3 is now Generally Available (GA) on {ocp-product-title} 4.14, 4.15, 4.16, and 4.17.
+
+[id="new-features-1-3_{context}"]
+== New features
+
+The following section highlights what is new in {builds-shortname} 1.3.
+
+* With this update, the {builds-operator} deploys Shipwright v0.14 APIs, controllers, and webhook.
+
+* With this update, while creating the `Build` or `BuildRun` resources with OpenShift Builds Client, you can set the parameters by using the `--param-value` flag.
+
+[id="fixed-issues-1-3_{context}"]
+== Fixed issues
+
+The following section highlights fixed issues in {builds-shortname} 1.3.
+
+* Before this update, when you installed {builds-shortname} without installing {pipelines-shortname}, the {builds-shortname} attempted to deploy {pipelines-shortname} and enable Pipelines controllers. {builds-shortname} incorrectly configured {pipelines-shortname} and failed to deploy the controllers. With this update, {builds-shortname} creates a valid `TektonConfig` custom resource and correctly deploys the Pipelines controllers.
+
+* Before this update, when you tried to run builds using {builds-title} installed on ARM64, IBM Power(R), or IBM Z(R) worker node CPU architectures, the build would fail because images were based on AMD64 platform only. Now, the AMD64 images are replaced with a `multi-arch` image in `buildah` and `source-to-image` build strategies, therefore you can run builds on all the architectures.
+
+* Before this update, when you enabled cluster monitoring in the `openshift-builds` namespace, the `TargetDown` alert was received on the Shared Resources Container Storage Interface (CSI) Driver `ServiceMonitor` resource due to an incorrect server name address. With this update, the server name is updated to the correct address in the Shared Resources CSI Driver `ServiceMonitor` resource and the `TargetDown` alert is no longer triggered.
+
+* Before this update, the Operator logo was not displayed in the OperatorHub. To display the Operator logo in the OperatorHub, you had to update the logo in the File Based Catalog (FBC) catalog format even after specifying it in the CSV file. With this update, the Operator logo data is specified in the catalog `config.yaml` files and the logo is displayed in the OperatorHub.
+
+[id="removed-features-1-3_{context}"]
+== Removed features
+
+The following section highlights removed features in {builds-shortname} 1.3.
+
+* With this update, the following channels are removed from {builds-shortname}:
```

Comment:
Hi @eromanova97, Sorry! we need little change here. We are not removing the channel in this release, instead we want to put a note for the user to upgrade.
"The following channels will be removed from builds-1.4, please upgrade to builds-1.3"